src/components/CardsDashboard/Search/Search.test.js

```diff
@@ -1,24 +1,19 @@
-import React from 'react';
-import { shallow } from 'enzyme';
 import { Search } from './Search';
 
 describe('<Search />', () => {
-  let wrapper;
-  const props = {
+  const defaultProps = {
     handleChange: jest.fn(),
   };
-  beforeEach(() => {
-    wrapper = shallow(<Search {...props} />);
-  });
+  const setup = buildSetup(Search, defaultProps, null);
 
   it('should render without crash', () => {
-    expect(wrapper).toMatchSnapshot();
+    expect(setup().wrapper).toMatchSnapshot();
   });
 
   it('should call onChange when typing in input', () => {
-    wrapper.find('input').simulate('change', { target: { value: '1' } });
-    wrapper.find('input').simulate('change', { target: { value: '2' } });
+    setup().wrapper.find('input').simulate('change', { target: { value: '1' } });
```

Comment:
Every time `setup` is called new component is mounted. This has minor impact on test performance.
Call `setup` only once in a test case.